### DISCUSSION TITLE
能否将txt的输出改为text/plain，去掉render_template_string方法里的html标签？

### DISCUSSION THREAD

**Discussion by wszhoho** [Guovin/iptv-api#736] [Ideas]
不去掉html标签的话，人眼看输出的网页是没问题，但是到了tvbox里，会滚动header，等不到更新时间出来，菜单就自动隐藏掉了。
亲测有效 
from flask import Flask, render_template_string,make_response
def show_txt():
    ret = get_result_file_content(file_type="txt")
    response = make_response(ret)
    response.mimetype = 'text/plain'
    return response

**Comment by Guovin** [comment]
收到，感谢你的提议，已采纳

**Comment by Guovin** [comment]
已实现，请更新
